src/cmds/test.ts

```diff
@@ -53,8 +60,15 @@ export const handler = (
     console.log(chalk.red('NO CODE FOUND'));
     return;
   }
+  if (exactTests){
+    exactTests = exactTests.sort((a,b) => a-b);
+    if (exactTests[0] < 1){
+      console.log('You should only insert positive integers at the exact tests flag numbers');
+      return;
+    }
+  }
   const tester: Tester = testers[testerOpt];
-  const tests: ProblemTests = readProblemTests(testsPath);
+  const tests: ProblemTests = readProblemTests(testsPath, exactTests);
   tester.beforeAll();
   const results = tests.map((test: ProblemTest, i) => tester.execute(`${i}`, test));
```

Comment:
Instead of changing `readProblemTests` you could just change this line. Call the execute only for the tests ids required. Also the results output should be handled so that we don't have a remapping of ids (for example you run tests 2 3 and they are show with ids 0 1 in results).